srtcore/buffer_tools.h

```diff
@@ -188,14 +187,53 @@ class CSndRateEstimator
         bool empty() const { return m_iPktsCount == 0; }
     };
 
-    int incSampleIdx(int val, int inc = 1) const;
+private:
+    static const int NUM_PERIODS        = 10;
+    static const int SAMPLE_DURATION_MS = 100; // 100 ms
 
     Sample m_Samples[NUM_PERIODS];
 
     time_point m_tsFirstSampleTime; //< Start time of the first sample.
     int        m_iFirstSampleIdx;   //< Index of the first sample.
     int        m_iCurSampleIdx;     //< Index of the current sample being collected.
     int        m_iRateBps;          //< Rate in Bytes/sec.
+
+    int incSampleIdx(int val, int inc = 1) const;
+};
+
+class CMobileRateEstimator:CSndRateEstimator
```

Comment:
Why is this class called "mobile" rate estimator? 🤔 
```suggestion
class CMobileRateEstimator
    : CSndRateEstimator
```

Reply:
Moving should be better. I wanted to remove the send/received flag because it's generic and the CSndRateEstimator fulfill, parse and empty, it's not a moving value.